### PR TITLE
Set forceId = false for AccessToken

### PR DESCRIPTION
**Pull request by jannyHou**
Connect to strongloop/loopback-datasource-juggler#849

Some test cases need refactor after we set forceId as true by default.
Details related please see: https://github.com/strongloop/loopback-datasource-juggler/pull/982#issuecomment-227941053

### CODE REVIEW

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@bajtos PTAL, thanks!

File: test/fixtures/user-integration-app/common/models/post.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "post",
+  "forceId": false,
```
Comment (bajtos) [inline test/fixtures/user-integration-app/common/models/post.json]: AFAICT, we are not setting `Post.id` explicitly in any of the existing tests. Why is it necessary to set `forceId` to `false` in that case?

File: test/relations.integration.js [hunk 2]
```diff
@@ -882,11 +882,15 @@ describe('relations - integration', function() {
         { properties: { name: 'string' },
           dataSource: 'db',
           plural: 'todo-lists',
+          forceId: false,
```
Comment (bajtos) [inline test/relations.integration.js]: ditto

File: test/relations.integration.js [hunk 3]
```diff
@@ -882,11 +882,15 @@ describe('relations - integration', function() {
...
         }
       );
       var todoItem = app.model(
         'todoItem',
-        { properties: { content: 'string' }, dataSource: 'db' }
+        { properties: { content: 'string' },
+          dataSource: 'db',
+          forceId: false,
```
Comment (bajtos) [inline test/relations.integration.js]: and here too.

BTW the indentation is off here too, both L891 an L892 are indented too much.

**Comment by bajtos** [comment]
@jannyHou I am confused why is it necessary to set `forceId:false` in test models, since I don't see any test that would be explicitly setting `id` when creating new instances. It does not match my mental model of how LoopBack & Juggler works.

Could you please elaborate little bit more on why are those changes necessary, so that I can understand what I am missing now?

**Comment by bajtos** [comment]
@jannyHou let's wait until #2551 is landed, I think we will be able to revert most of the changes proposed here.

**Comment by bajtos** [comment]
With #2551 landed, there is only one test needing `forceId:false`. I am going to force-push into this pull request the updated version that works for me. For posterity, @jannyHou's original commit was 8553c2d.

**Comment by jannyHou** [comment]
@bajtos Thank you 👍  and I will take a look into the CI failings

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
The failing ones are caused by authentication in some test cases.
For example:
```
Wed, 27 Jul 2016 21:45:05 GMT strong-error-handler Handling Error: Authorization Required
    at C:\jenkins\workspace\loopback\b88abec9\lib\application.js:393:21
    at C:\jenkins\workspace\loopback\b88abec9\lib\model.js:318:7
    at C:\jenkins\workspace\loopback\b88abec9\common\models\acl.js:471:23
    at C:\jenkins\workspace\loopback\b88abec9\node_modules\async\lib\async.js:251:17
    at done (C:\jenkins\workspace\loopback\b88abec9\node_modules\async\lib\async.js:132:19)
    at C:\jenkins\workspace\loopback\b88abec9\node_modules\async\lib\async.js:32:16
    at C:\jenkins\workspace\loopback\b88abec9\node_modules\async\lib\async.js:248:21
    at C:\jenkins\workspace\loopback\b88abec9\node_modules\async\lib\async.js:572:34
    at C:\jenkins\workspace\loopback\b88abec9\common\models\acl.js:453:17
    at C:\jenkins\workspace\loopback\b88abec9\common\models\role.js:269:21
    at C:\jenkins\workspace\loopback\b88abec9\node_modules\continuation-local-storage\node_modules\async-listener\glue.js:188:31
    at process._tickDomainCallback [as _tickCallback] (node.js:492:13)
<testcase classname="loopback.rest" name="should report 200 for GET /:id/exists found" time="0.007"/>
Wed, 27 Jul 2016 21:45:05 GMT strong-error-handler Response status 401 data {"statusCode":401,"name":"Error","message":"Authorization 
```

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@slnode test please

**Comment by jannyHou** [comment]
I am landing it since all loopback CI are green :squirrel:

**Pull request merged**